data/en/cfchartdata.json

```diff
@@ -14,5 +14,14 @@
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/tag/cfchartdata"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/tag/cfchartdata"}
 	},
-	"links": []
-}
+	"links": [],
+	"examples": [
+		{
+			"title":"CFML Markup",
```

Comment:
Ideally let's use a short descriptive title, like "Simple Bar Chart" rather than using "CFML Markup" for each.